spring-cloud-dataflow-admin-starter/src/main/java/org/springframework/cloud/dataflow/admin/controller/SecurityController.java

```diff
@@ -31,10 +32,11 @@
  */
 @RestController
 @RequestMapping("/security/info")
+@EnableConfigurationProperties(SecurityProperties.class)
 public class SecurityController {
 
 	@Autowired
-	private Environment environment;
+	SecurityProperties securityProperties;
```

Comment:
why is this not private? also, can we go ahead and switch away from field injection while we're making changes here?